src/custom-typings.d.ts

```diff
@@ -43,9 +43,6 @@ import * as _ from 'lodash'
  *
  */
 
-// support NodeJS modules without type definitions
-declare module '*';
```

Comment:
For some reason I have got compiler error with that line. I have checked now and not true anymore...
I have just removed this change from the pull request (so now the wildcar declaration is there).